std/range/package.d

```diff
@@ -5217,54 +5217,54 @@ auto sequence(alias fun, State...)(State args)
 
 // iota
 /**
-   Construct a range of values that span the given starting and stopping
-   values.
 
-   Params:
-   begin = The starting value.
-   end = The value that serves as the stopping criterion. This value is not
-        included in the range.
-   step = The value to add to the current value at each iteration.
+Construct a range of values that span the given starting and stopping values.
 
-   Returns:
-   A range that goes through the numbers $(D begin), $(D begin + step),
-   $(D begin + 2 * step), $(D ...), up to and excluding $(D end).
+Params:
+begin = The starting value.
+end = The value that serves as the stopping criterion. This value is not
+    included in the range.
+step = The value to add to the current value at each iteration.
 
-   The two-argument overloads have $(D step = 1). If $(D begin < end && step <
-   0) or $(D begin > end && step > 0) or $(D begin == end), then an empty range
-   is returned. If $(D step == 0) then $(D begin == end) is an error.
+Returns:
+A range that goes through the numbers $(D begin), $(D begin + step), $(D begin +
+2 * step), $(D ...), up to and excluding $(D end).
 
-   For built-in types, the range returned is a random access range. For
-   user-defined types that support $(D ++), the range is an input
-   range.
+The two-argument overloads have $(D step = 1). If $(D begin < end && step < 0)
+or $(D begin > end && step > 0) or $(D begin == end), then an empty range is
+returned. If $(D step == 0) then $(D begin == end) is an error.
 
-    Example:
-    ---
-    void main()
-    {
-        import std.stdio;
+For built-in types, the range returned is a random access range. For
+user-defined types that support $(D ++), the range is an input
+range.
+
+Example:
+---
+void main()
+{
+    import std.stdio;
 
-        // The following groups all produce the same output of:
-        // 0 1 2 3 4
+    // The following groups all produce the same output of:
+    // 0 1 2 3 4
 
-        foreach (i; 0 .. 5)
-            writef("%s ", i);
-        writeln();
+    foreach (i; 0 .. 5)
+        writef("%s ", i);
+    writeln();
 
-        import std.range : iota;
-        foreach (i; iota(0, 5))
-            writef("%s ", i);
-        writeln();
+    import std.range : iota;
+    foreach (i; iota(0, 5)) // alternatively: iota(5)
+        writef("%s ", i);
+    writeln();
 
-        writefln("%(%s %|%)", iota(0, 5));
+    writefln("%(%s %|%)", iota(0, 5));
 
-        import std.algorithm.iteration : map;
-        import std.algorithm.mutation : copy;
-        import std.format;
-        iota(0, 5).map!(i => format("%s ", i)).copy(stdout.lockingTextWriter());
-        writeln();
-    }
-    ---
+    import std.algorithm.iteration : map;
+    import std.algorithm.mutation : copy;
+    import std.format;
+    iota(0, 5).map!(i => format("%s ", i)).copy(stdout.lockingTextWriter());
+    writeln();
+}
+---
```

Comment:
The original doc style (and the prevalent style in this particular file) has the doc flush with the symbol being documented. I see reason to take the opportunity to consolidate style a bit, and as a contributor to the docs I find the gratuitous indent unpleasant.